docs/Home.md

```diff
@@ -0,0 +1,38 @@
+# Home
```

Comment:
Can we merge this with the index page? Because I feel like there's a lot of overlap where both just mostly provide links to other pages.

Reply:
I'll take a look

Reply:
I think the Home.md as the new Readme should be enough then, since it's the more complete version